src/templates/.eslintrc.js

```diff
@@ -0,0 +1,55 @@
+const { resolve } = require('path')
```

Comment:
As we move to ESM world, it could be a good idea to rename all ESLint config files to `.cjs` as we go